platform/_partials/install/upgrade.mdx

```diff
@@ -1,7 +1,14 @@
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
-Upgrade vCluster Platform via:
+## Upgrade the platform
```

Comment:
Why are we using this section to upgrade the platform in the install docs? Why isn't this being used to install the platform?

Reply:
As a new user this doesn't make sense to me.

Reply:
Capturing for later: https://linear.app/loft/issue/DOC-301/air-gapped-install-improvements